specifications/api/swagger.yaml

```diff
@@ -723,12 +723,26 @@ x-stakePoolMargin: &stakePoolMargin
 
       May be omitted if the wallet hasn't found the pool's registration cerificate yet.
 
+x-jormungandrStakePoolSaturation: &jormungandrStakePoolSaturation
+  type: number
+  minimum: 0
+  description: |
+    Saturation-level of the pool based on the desired number of pools aimed by the network.
+    A value above `1` indicates that the pool is saturated.
+  example: 0.74
+
 x-stakePoolSaturation: &stakePoolSaturation
   type: number
   minimum: 0
   description: |
     Saturation-level of the pool based on the desired number of pools aimed by the network.
     A value above `1` indicates that the pool is saturated.
+
+    The `non_myopic_member_rewards` take oversaturation into account, as specified by the [specs](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/delegationDesignSpec/latest/download-by-type/doc-pdf/delegation_design_spec).
+
+    The saturation is based on the live `relative_stake`. The saturation at the end of epoch e,
+    will affect the rewards paid out at the end of epoch e+3.
```

Comment:
:+1:

Reply:
I've inferred these, but not checked
- [ ] `The saturation is based on the live `relative_stake`
- [ ] `will affect the rewards paid out at the end of epoch e+3`

Reply:
> will affect the rewards paid out at the end of epoch e+3

Well, based on:

<img width="665" alt="Skärmavbild 2020-09-22 kl  13 58 40" src="https://user-images.githubusercontent.com/304423/93879363-de8c5e80-fcdb-11ea-8485-0ecb131bf5e0.png">

> The saturation is based on the live relative_stake

Looks pretty clear from a comment in #1854 : 

> This is not based on any snapshot, but uses the current ledger state.

And I noticed differences in ranking when trying out live.